setup.py

```diff
@@ -26,6 +26,7 @@
     "torchtyping",
     "tqdm",
     "CLIP",
+    "transformers==4.40.1"
```

Comment:
@jonasd4 @a1247418 Should we keep `transformers` as a `thingsvision` dependency?

Reply:
I would vote for keeping it! Easier to add other models in the future and the people do not have to install something extra!

Reply:
@jonasd4 Let's change to "transformers==4.40.*"?